js/components/tabIcon.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Just noticed this now- the new preferred location for components would be in:
`app/renderer/components` :smile: 

This lets us easily break up the rendering portion (client side JavaScript) and the browser portion (which has access to node / makes use of muon (electron))